plugin/template/README.md

```diff
@@ -31,6 +32,8 @@ template CLASS TYPE [ZONE...] {
   in a response with an empty answer section.
 * `rcode` **CODE** A response code (`NXDOMAIN, SERVFAIL, ...`). The default is `NOERROR`. Valid response code values are
   per the `RcodeToString` map defined by the `miekg/dns` package in `msg.go`.
+* `ederror` **EXTENDED_ERROR_CODE** is an extended DNS error code as a number defined in `RFC8914` (0, 1, 2,..., 24).
+              **EXTRA_REASON** is an additional string explaining reason for receiving given extended DNS error.
```

Comment:
```suggestion
              **EXTRA_REASON** is an additional string explaining the reason for returning the error.
```

Reply:
thanks for the review! adjusted as you suggested